01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+for (let i = 1; i <= 20; i++) {
+  if (i % 5 === 0 && i % 3 === 0) {
+    console.log("Fizzbuzz");
```

Comment:
Fizzbuzz ではなく FizzBuzz です。

```suggestion
    console.log("FizzBuzz");
```